common/state/sebio.cpp

```diff
@@ -0,0 +1,308 @@
+/* Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "sebio.h"
+#include "crypto.h"
+#include "state.h"
+#include "types.h"
+#include "crypto.h"
+#include "c11_support.h"
+#include "error.h"
+
+#ifdef DEBUG
+    #define SAFE_LOG(LEVEL, FMT, ...) Log(LEVEL, FMT, ##__VA_ARGS__)
+#else // DEBUG not defined
+    #define SAFE_LOG(LEVEL, FMT, ...)
+#endif // DEBUG
+
+//TODO move definitions to makefiles
+
+//#ifdef SEBIO_NO_DEFAULT
+//    #define SEBIO_ON_BIOX
+//#else //SEBIO_NO_DEFAULT not defined
+    #define SEBIO_ON_BLOCK_STORE
+//#endif //SEBIO_NO_DEFAULT
```

Comment:
Not sure what's going on with these #define's but it looks suspicious

Reply:
fixed